'imagetag' -> 'image' in edit set command examples

demobox opened the pull request:
'imagetag' is not/no longer supported as a command

```release-note
Correct a reference to a not/no longer used kustomize subcommand in the documentation
```

demobox committed:
'imagetag' -> 'image' in edit set command examples

'imagetag' is not/no longer supported as a command

demobox commented:
I signed it

hpandeycodeit commented:
@demobox Thanks for the PR! Can you edit the PR to add the Release Notes Section and the Kind of fix this PR is providing?

https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md

demobox commented:
I signed it

pwittrock commented:
/lgtm

pwittrock commented:
/approve

pwittrock commented:
/oktotest

seans3 commented:
/approve

seans3 commented:
```release-note
NONE
```

seans3 commented:
/kind cleanup
/priority important-soon

seans3 reviewed (approved):
Please add a release note--thanks.

seans3 commented:
/ok-to-test

demobox commented:
> Can you edit the PR to add the Release Notes Section and the Kind of fix this PR is providing?

@hpandeycodeit Added - thanks for the hint!

@seans3 I see you also added `NONE` as the release note - was that intended as a temporary value to move the process along? Happy to update the note I added in the PR description if we feel `NONE` is more appropriate for this change
